java-squid/src/main/java/org/sonar/java/model/VisitorsBridge.java

```diff
@@ -116,14 +117,14 @@ public void visitFile(@Nullable AstNode astNode) {
       } else {
         SemanticModel.handleMissingTypes(tree);
       }
-      JavaFileScannerContext context = new DefaultJavaFileScannerContext(tree, (SourceFile) getContext().peekSourceCode(), getContext().getFile(), semanticModel, analyseAccessors);
-      for (JavaFileScanner scanner : scanners) {
-        scanner.scanFile(context);
-      }
-      if (semanticModel != null) {
-        // Close class loader after all the checks.
-        semanticModel.done();
-      }
+    }
+    JavaFileScannerContext javaFileScannerContext = new DefaultJavaFileScannerContext(tree, (SourceFile) getContext().peekSourceCode(), getContext().getFile(), semanticModel, analyseAccessors);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 193 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00103)